core/templates/pages/exploration-player-page/services/content-translation-manager.service.spec.ts

```diff
@@ -322,6 +325,12 @@ describe('Content translation manager service', () => {
       writtenTranslations.addWrittenTranslation(
         'ca_1', 'fr', 'unicode', 'fr 2');
 
+      // This throws "Type '"DummyInteraction"' is not assignable to type
+      // '"ImageClickInput" | "NumberWithUnits" | "NumericExpressionInput" |
+      // "NumericInput" | "DragAndDropSortInput" | "ItemSelectionInput" |
+      // "Continue" | "GraphInput" | "EndExploration" | ... 11 more ... |
+      // "RatioExpressionInput"'.""
```

Comment:
```suggestion
      // This throws "Type '"DummyInteraction"' is not assignable to type
      // '"ImageClickInput" | "NumberWithUnits" | "NumericExpressionInput" |
      // "NumericInput" | "DragAndDropSortInput" | "ItemSelectionInput" |
      // "Continue" | "GraphInput" | "EndExploration" | ... 11 more ... |
      // "RatioExpressionInput"'." We need to suppress this error because
      // we need to change the value of 'interaction.id' for testing
      // purposes.
```